Sources/RemoteNotification.swift

```diff
@@ -88,10 +99,27 @@ public struct RemoteNotification: CustomStringConvertible, Equatable {
         /// The localization arguments for the title shown on the Apple Watch. Derived from userInfo["aps"]["alert"]["title-loc-args"]
         public let wearableTitleLocalizationArguments: [String]?
         
-        init?(remoteNotification: [String : AnyObject]) {
+        // MARK: Equatable
+        
+        public static func ==(lhs: Alert, rhs: Alert) -> Bool {
+            return lhs.body == rhs.body
+                && lhs.bodyLocalizationKey == rhs.bodyLocalizationKey
+                && lhs.bodyLocalizationArguments ?? []  == rhs.bodyLocalizationArguments ?? [] 
+                && lhs.actionLocalizationKey == rhs.actionLocalizationKey
+                && lhs.launchImageName == rhs.launchImageName
+                && lhs.wearableTitle == rhs.wearableTitle
+                && lhs.wearableTitleLocalizationKey == rhs.wearableTitleLocalizationKey
+                && lhs.wearableTitleLocalizationArguments ?? [] == rhs.wearableTitleLocalizationArguments ?? []
+        }
+        
+        init?(remoteNotification: [String : Any]) {
             // Alert is represented as either a dictionary or a single string.
-            guard let alert = remoteNotification[APSServiceKey]?[alertKey] as? [String : AnyObject] else {
-                guard let body = remoteNotification[APSServiceKey]?[alertKey] as? String else {
+            guard let apnsDictionary = remoteNotification[apsServiceKey] as! [String : Any]? else {
```

Comment:
I think a crash is reasonable in these cases. Basically if apple changed what they put in these dictionaries, leaving stuff as `as?` would basically just make everything start failing silently... I think a compromise may be having a function that does this pattern for us and we can control how it blows up.